app/main.py

```diff
@@ -1,8 +1,47 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
```

Comment:
after the opening bracket, write all arguments on a new line, including the first one